student.rb

```diff
@@ -0,0 +1,12 @@
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(id, age, parent_permision, classroom, name = 'unknown')
+    super(id, name, age, parent_permision)
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=parent_permission%20as%20parameter.-,name%20and%20parent_permission%20are%20optional%20and%20have%20default%20values%20of%20%22Unknown%22%20and%20true.,-Getters%20for%20%40id), The `parent_permission` parameter has a default value `true`. Please add it.

- [ ] [Optional] [Highly recommended] If you implement the `id` suggestion above, please remove the `id` from lines 4 and 5.

[Hint] It should be like that after fixing:
```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```